easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2rc2-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,19 @@
+name = 'MVAPICH2'
+version = '2.2rc2'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = "This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH."
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/']
+sources = [SOURCELOWER_TAR_GZ]
+
+osdependencies = [('libibverbs-dev','libibverbs-devel'),('libibmad-dev','libibmad-devel')]
```

Comment:
@boegel, this is how I approached it on a system I am installing at the moment. I don't necessarily know the ubuntu packages per say, but the `libibverbs-devel` and `libibmad-devel`

From my perspective, I either use the packages from Mellanox OFED, or the Intel OFED, as they can be tuned for their various cards. But nevertheless, the distros will have the same packages available, and the OpenMPI is having the dependancies in the same way as well.

So whether you use the distro packages, or the OFED provided by the relevant vendors, then this dependancy should work.

The only thing with the osdependancies is that we still have to go away, and use the relevant OS package management to install them, it would be neat to get easybuild to install these for you. But this might already be the case, as I am pretty new to easybuild